src/main/resources/templates/db_scripts/schema.json

```diff
@@ -928,7 +928,7 @@
     {
       "run": "after",
       "snippetPath": "oaipmh/createOaiPmhViewFunction.sql",
-      "fromModuleVersion": "19.2.5"
+      "fromModuleVersion": "19.3.0"
```

Comment:
Given that the next mainline release is going to be 19.4.0 or 20.0.0, should this be 19.4.0 rather than 19.3.0?

Reply:
We should use 19.3.1 because all versions >= 19.3.1 already have run the .sql file.
Otherwise upgrading from 19.3.1 to 19.4.0 will re-run the file - this does not cause any harm because it is unneccessary.

Reply:
Ok. I dislike this workaround but I understand and accept it. Thanks @julianladisch for the explanation